### ISSUE TITLE
No module named 'gseapy.gse'

### ISSUE THREAD

**Issue by ShriHarshaAdapalaThirumala** [zqfang/GSEApy#214]
### Setup

I am reporting a problem with GSEApy version, Python version, and operating
system as follows:

```python
import sys; print(sys.version)
import platform; print(platform.python_implementation()); print(platform.platform())

3.11.4 | packaged by conda-forge | (main, Jun 10 2023, 18:08:41) [Clang 15.0.7 ]
CPython
macOS-13.4.1-arm64-arm-64bit

```

(*Please copy and run the above in your Python, and copy-and-paste the output*)

### Expected behaviour

import gseapy as gp

Should get import without any issue 

### Actual behaviour

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[57], line 1
----> 1 import gseapy as gp

File [~/Documents/programming_environments/bio_env/lib/python3.11/site-packages/gseapy/__init__.py:9](https://file+.vscode-resource.vscode-cdn.net/Users/shri/Documents/Bioinformatics/~/Documents/programming_environments/bio_env/lib/python3.11/site-packages/gseapy/__init__.py:9)
      7 from .biomart import Biomart
      8 from .enrichr import Enrichr
----> 9 from .gsea import GSEA, Prerank, Replot, SingleSampleGSEA
     10 from .parser import get_library_name
     11 from .plot import barplot, dotplot, gseaplot, heatmap

File [~/Documents/programming_environments/bio_env/lib/python3.11/site-packages/gseapy/gsea.py:15](https://file+.vscode-resource.vscode-cdn.net/Users/shri/Documents/Bioinformatics/~/Documents/programming_environments/bio_env/lib/python3.11/site-packages/gseapy/gsea.py:15)
     12 import pandas as pd
     14 from gseapy.base import GSEAbase
---> 15 from gseapy.gse import Metric, gsea_rs, prerank_rs, ssgsea_rs  # import gseapy rust lib
     16 from gseapy.parser import gsea_cls_parser
     17 from gseapy.plot import gseaplot

ModuleNotFoundError: No module named 'gseapy.gse'

### Steps to reproduce

Just imported.

**Comment by zqfang** [comment]
please use pip install. Bioconda don't have py311 build yet.

```shell
pip install gseapy
```

**Comment by ShriHarshaAdapalaThirumala** [comment]
On trying to install via pip I am getting following error:

***@***.***

From: Zhuoqing Fang ***@***.***>
Date: Wednesday, July 12, 2023 at 2:26 PM
To: zqfang/GSEApy ***@***.***>
Cc: Shri Harsha Adapala Thirumala ***@***.***>, Author ***@***.***>
Subject: Re: [zqfang/GSEApy] No module named 'gseapy.gse' (Issue #214)

please use pip install. Bioconda don't have py311 build yet.

pip install gseapy

**Comment by ShriHarshaAdapalaThirumala** [comment]
> please use pip install. Bioconda don't have py311 build yet.
> 
> ```shell
> pip install gseapy
> ```

Getting following error on trying to install via pip

<img width="1440" alt="image" src="https://github.com/zqfang/GSEApy/assets/130099379/aa8c2af3-d9a5-460c-a0ae-d30753c687ba">

**Comment by zqfang** [comment]
I'm sorry, I just notice you're using "ARM-Macos". I don't have a M1-chip Mac to build the binary.

In this case, you have to install rust additionally, see here:  https://www.rust-lang.org/tools/install

**Comment by zqfang** [comment]
@mari-ga, uninstall the gseapy and run the code below

```shell
curl https://sh.rustup.rs -sSf | sh -s -- -y
# export rust compiler
export PATH="$PATH:$HOME/.cargo/bin"
# install gseapy
pip install gseapy
```

**Comment by Yecats77** [comment]
Excuse me. I have tried the methods you mentioned above,
```sh
curl https://sh.rustup.rs -sSf | sh -s -- -y
. "$HOME/.cargo/env"  
export PATH="$PATH:$HOME/.cargo/bin"
pip install gseasy
```
and got an error
```txt
ERROR: Could not find a version that satisfies the requirement gseasy (from versions: none)
ERROR: No matching distribution found for gseasy
```
My system info
```txt
3.9.18 (main, Sep 11 2023, 13:41:44) 
[GCC 11.2.0]
CPython
Linux-5.4.0-165-generic-x86_64-with-glibc2.31
```
Could you please help with this installation issue? Thank you.

**Comment by zqfang** [comment]
You shoud use
```
pip install gseapy
```

Not `gseasy`

Zhuoqing
From: 77 ***@***.***>
Date: Thursday, July 18, 2024 at 08:14
To: zqfang/GSEApy ***@***.***>
Cc: Zhuoqing Fang ***@***.***>, Comment ***@***.***>
Subject: Re: [zqfang/GSEApy] No module named 'gseapy.gse' (Issue #214)

Excuse me. I have tried the methods you mentioned above,

curl https://sh.rustup.rs -sSf | sh -s -- -y

. "$HOME/.cargo/env"

export PATH="$PATH:$HOME/.cargo/bin"

pip install gseasy

and got an error

ERROR: Could not find a version that satisfies the requirement gseasy (from versions: none)

ERROR: No matching distribution found for gseasy

My system info

3.9.18 (main, Sep 11 2023, 13:41:44)

[GCC 11.2.0]

CPython

Linux-5.4.0-165-generic-x86_64-with-glibc2.31

Could you please help with this installation issue? Thank you.

**Comment by Yecats77** [comment]
Yes. I realized that it was my mistake, and the issue was not actually valid. As soon as I recognized this, I proceeded to delete the issue from the repository.

Sorry for any confusion or inconvenience.
And thank you for developing the nice tool!
________________________________
